local-llm/setup.py

```diff
@@ -33,10 +33,10 @@
     ],
     entry_points={
         'console_scripts': [
-            'llm = llm:cli',
+            'local-llm = local_llm:cli',
```

Comment:
This hurts me but seems correct :)

Reply:
lol yeah 😅 

Using `-` in the command line tool felt better than `_`, but it seems to be [the opposite case for the python module name](https://stackoverflow.com/questions/761519/is-it-ok-to-use-dashes-in-python-files-when-trying-to-import-them) - fun fun fun